terraform/environments/xhibit-portal/locals.tf

```diff
@@ -38,5 +38,5 @@ locals {
   # environment specfic variables
   # example usage:  
   # example_data = local.application_data.accounts[local.environment].example_var
-  application_data = fileexists("./application_variables.json") ? jsondecode(file("./application_variables.json")) : []
+  application_data = fileexists("./application_variables.json") ? jsondecode(file("./application_variables.json")) : {}
```

Comment:
I hope this will not re-start xhibit onboarding. If yes, then I guess reverting would not make any difference.

Reply:
No this shouldn't have any impact